src/V3UndrivenCapture.cpp

```diff
@@ -0,0 +1,212 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Capture task/function write summaries for undriven checks
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2025 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#include "V3UndrivenCapture.h"
+
+#include "V3Error.h"
+#include "V3Global.h"
+
+VL_DEFINE_DEBUG_FUNCTIONS;
+
+namespace {
+
+struct Stats final {
+    uint64_t ftasks = 0;
+    uint64_t varWrites = 0;
+    uint64_t callEdges = 0;
```

Comment:
I think can be moved into the Visitor if print on destruction.  Should these be V3Stats instead so go into __stats.txt, or you think only really needed for debug?   Not sure they are needed at all once you have algorithm solid.